### PR TITLE
Adding the first web page

### LINKED ISSUE #6: Add the first web page
**Requirements:**

1. Altering the 'views.py' file and add a function to render our first web page.
2. Include the route of the function in the 'urls.py' file.
3. Add a static web page template.

### PR DESCRIPTION
**Pull request by tamirho**
1. Created a new app directory called 'main' and register it by modifying the settings.py file.
2. Created a new function to render our home page in 'views.py' and created a new route in the 'urls.py' file.
3. Added a static home page template.
4. Modifed the .gitignore file in order to ignore nohup.out files.

fix #6

### CODE REVIEW

**Review by OrAsaf — APPROVED** [review]
LGTM

**Comment by simzacks** [comment]
If you look at the files in https://github.com/beyond-io/roo.me/pull/12/files
both home.html and views.py have a note: "No newline at end of file"
When we add flake8 this will actually be flagged as an error, so its worth fixing now.

**Comment by tamirho** [comment]
I fixed Sim's suggestion and squashed it into one commit.

File: main/templates/main/home.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            text-align: center;
+            padding: 100px;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="content">
+        <img src="https://user-images.githubusercontent.com/53123142/110364209-287bbd80-804c-11eb-91c3-a158b115ef27.png"> </img>
+        <h1>Comming Soon</h1>
```
Comment (DanielMy777) [inline main/templates/main/home.html]: There's a double 'm' typo here

**Review by DanielMy777 — CHANGES_REQUESTED** [review]
I Loved the web page! :)
Please see my small comment.

File: main/templates/main/home.html (hunk 1 above)
Comment (tamirho) [reply main/templates/main/home.html]: Thanks for the attention

**Review by simzacks — APPROVED** [review]
+1

**Pull request merged**